### ISSUE TITLE
More options for emissive.properties

### ISSUE THREAD

**Issue by LavidayFlaver** [sp614x/optifine#6170]
Here is my list of ideas:
1. Add a feature where instead of the texture being rendered at full brightness, it can have its light value multiplied, divided, subtracted, or added to.
2. Add a feature where an emissive texture can give off dynamic lighting.
3. Add a feature for multiple suffixes which can be configured individually.

**Comment by DanielCehic** [comment]
While I agree that it'd be more convenient to be able to do this through a config file, I'll just throw in that you actually can adjust how bright an emissive texture is by adjusting it's transparency.

**Comment by LavidayFlaver** [comment]
> While I agree that it'd be more convenient to be able to do this through a config file, I'll just throw in that you actually can adjust how bright an emissive texture is by adjusting it's transparency.

I had no idea. Thanks for letting me know!

**Comment by DanielCehic** [comment]
Ah, forgive me. It's been a while since I played around with this. Let me rephrase my statement:

Changing the opacity of an emissive texture does indeed change how bright they are, but for whatever reason it only seems to work on items, but not on blocks and armors (and I assume entities as well). [Here](https://imgur.com/a/qCcQMeW) is a practical example of it in action.
On the left we have a Bottle o' Enchanting from Vanilla Tweaks with a custom half-opacity emissive texture made by myself.
In the middle we have a custom Warped Mushroom from Mickey Joe's RID with full opacity emissives.
And on the right, we have a placed Enchanting Table (custom model also from RID) with half-opacity emissives made by myself, except that it doesn't work since it is a block and not an item. It also doesn't work on the table when in item form.

Hopefully that should eliminate any confusion I caused. Tbh I'm not even sure if you're even meant to be able to do this in the first place.
